Source/FolioReaderPage.swift

```diff
@@ -656,7 +663,11 @@ extension UIWebView {
     // MARK: - Create and show menu
     
     func createMenu(options options: Bool) {
-        isShare = options
+		guard readerConfig.useReaderMenuController else {
+			return
```

Comment:
I don't think so: `UIMenuController` is a singleton which is owned by the app - not the framework. So this menu items will also effect the rest of the app which inlcudes Folio Reader. `createMenu()` sets `menuItems` to a new array and therefore removes all other menu items (also from the app). This should only happen if `useReaderMenuController` is set to `true`.